lib/db.js

```diff
@@ -63,6 +63,10 @@ function getMongoConnectionUrl(cb) {
     return mongoConnectionStringOS2(cb);
   } else if ("openshift3" === process.env.FH_MBAAS_TYPE) {
     return mongoConnectionStringOS3(cb);
+  } else if (process.env.FH_USE_LOCAL_DB) {
+    // Backwards compatible, hard coded localhost for local development
+    process.env.FH_MONGODB_CONN_URL = 'mongodb://localhost:27017';
```

Comment:
@aidenkeating yes, i'll change this to FH_LOCAL to avoid it using the admin database